src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -653,6 +653,28 @@ def is_mapper_device(device_name):
     return device_name.startswith(('/dev/mapper', '/dev/dm-'))
 
 
+def is_writable_raw_device(disk_path):
+    '''
```

Comment:
we don't use single quotes for docstrings anywhere in ceph-volume, please use double quotes: `"""` instead

Reply:
fixed. thx.